internal/static/metrics/agent.go

```diff
@@ -19,12 +19,14 @@ import (
 var DefaultConfig = Config{
 	Global:                 instance.DefaultGlobalConfig,
 	InstanceRestartBackoff: 5 * time.Second,
-	WALDir:                 "data-alloy/",
-	WALCleanupAge:          12 * time.Hour,
-	WALCleanupPeriod:       30 * time.Minute,
-	ServiceConfig:          cluster.DefaultConfig,
-	ServiceClientConfig:    client.DefaultConfig,
-	InstanceMode:           instance.DefaultMode,
+	// The following legacy WALDir path is intetionally kept for config conversion from static to Alloy.
+	// Consult Alloy maintainers for changes.
+	WALDir:              "data-alloy/",
```

Comment:
We should keep this line as `data-agent` like Robert said in his previous comment

Reply:
Oops, forgot this one. Done in the HEAD commit :D